packages/mwp-api-proxy-plugin/src/util/send.js

```diff
@@ -20,6 +19,10 @@ import config from 'mwp-cli/src/config';
 import { API_PROXY_PLUGIN_NAME } from '../config';
 
 export const API_META_HEADER = 'X-Meta-Request-Headers';
+const MEMBER_COOKIE_NAME =
+	process.env.NODE_ENV === 'production' ? 'MEETUP_MEMBER' : 'MEETUP_MEMBER_DEV';
```

Comment:
obnoxious nitpick would be nice if these would more const based but nice to have